### ISSUE TITLE
Kaltex rendering not working

### ISSUE THREAD

**Issue by pritesha** [benweet/stackedit.js#9]
Hi , 

I am unable to render Kaltex, it is shown as plain text. 
Other markdowns such as bold/italic are working correctly. 
`    const stackedit = new Stackedit();
    const divEl = document.getElementById('test11');
    stackedit.openFile({
      name: 'Filename',
      content: {
        text: '$$e^{-t}$$',
        properties: {
          extensions: {
            katex: {
              enabled: true,
            }
          }
        }
      }
    }, true /* silent mode */);

    // In silent mode, the `fileChange` event is emitted only once.
    stackedit.on('fileChange', (file) => {
      divEl.innerHTML = file.content.html;
    });
  }`

am i missing something ?

**Comment by benweet** [comment]
Duplicate of https://github.com/benweet/stackedit.js/issues/5. See https://github.com/benweet/stackedit.js/issues/5#issuecomment-385112904 for a workaround.
